Sources/Charcoal/Filters/Map/MapFilterViewController.swift

```diff
@@ -46,9 +66,10 @@ public class MapFilterViewController: FilterViewController {
 
     private let bboxFilter: Filter?
     private let polygonFilter: Filter?
+    private let radiusFilter: Filter?
 
     public init(title: String, latitudeFilter: Filter, longitudeFilter: Filter, radiusFilter: Filter,
-                locationNameFilter: Filter, bboxFilter: Filter?, polygonFilter: Filter?, selectionStore: FilterSelectionStore) {
+                locationNameFilter: Filter, bboxFilter: Filter?, polygonFilter: Filter?, defaultMode: MapFilterMode = .radius, selectionStore: FilterSelectionStore) {
```

Comment:
Can we change the param name to `defaultMapMode` so it's easy to see what it means on caller side?